hack/jenkins/verify-dockerized.sh

```diff
@@ -29,6 +29,17 @@ retry() {
 # This script is intended to be run from kubekins-test container with a
 # kubernetes repo mapped in. See k8s.io/test-infra/scenarios/kubernetes_verify.py
 
+REMOTE="$(git remote)"
+
+if [[ ${REMOTE} == *"bootstrap-upstream"* ]]; then
```

Comment:
hmm I see why we have this but I dislike this if/else :confused:

Reply:
might as well inline the `$(git remote)` here since we don't seem to reference it elsewhere

Reply:
I'll drop this if/else after we ditch the existing job.